notifier/chatters.rst

```diff
@@ -135,6 +135,120 @@ The result will be something like:
 
     The `field()` method was introduced in Symfony 5.1.
 
+Adding a header to a Slack Message
+-------------------------------------------
+
+To add a header to your message you can use the
+:class:`Symfony\\Component\\Notifier\\Bridge\\Slack\\Block\\SlackHeaderBlock` class::
+
+    use Symfony\Component\Notifier\Bridge\Slack\Block\SlackDividerBlock;
+    use Symfony\Component\Notifier\Bridge\Slack\Block\SlackHeaderBlock;
+    use Symfony\Component\Notifier\Bridge\Slack\Block\SlackSectionBlock;
+    use Symfony\Component\Notifier\Bridge\Slack\SlackOptions;
+    use Symfony\Component\Notifier\Message\ChatMessage;
+
+    $chatMessage = new ChatMessage('Symfony Feature');
+
+    $options = (new SlackOptions())
+        ->block((new SlackHeaderBlock('My Header')))
+        ->block((new SlackSectionBlock())->text('My message'))
+        ->block(new SlackDividerBlock())
+        ->block(
+            (new SlackSectionBlock())
+                ->field('*Max Rating*')
+                ->field('5.0')
+                ->field('*Min Rating*')
+                ->field('1.0')
+        );
+
+    // Add the custom options to the chat message and send the message
+    $chatMessage->options($options);
+
+    $chatter->send($chatMessage);
+
+The result will be something like:
+
+.. image:: /_images/notifier/slack/slack-header.png
+   :align: center
+
+.. versionadded:: 5.3
+
+    The `SlackHeaderBlock` class was introduced in Symfony 5.3.
```

Comment:
Double backticks needed

Reply:
Yes, I fixed several of these single back tick occurrences while merging. Thanks.

Reply:
Thank you

Reply:
Thank you Javier, and thank you Oskar for letting me know I'll double check in the future 👍

Reply:
Thanks for contributing to the docs 😎👍🏻